relay/network_graph/payment_path.py

```diff
@@ -7,4 +7,5 @@ class PaymentPath:
     fee: int
     path: List
     value: int
+    sender_pays_fees: bool = attr.ib(default=True)
```

Comment:
I dont think it is a good idea to have this as default. As you can see this already created the problem that you forgot further up to fill it with the right value.